maintainers/all-owners.sh

```diff
@@ -25,7 +25,16 @@ all_owners_raw() {
     else
       filter='.approvers'
     fi
-    curl -s "https://raw.githubusercontent.com/metal3-io/$repo/main/OWNERS" | \
+    if [ "$repo" = "hardware-classification-controller" ] ||  [ "$repo" = "ironic-client" ] \
+      || [ "$repo" = "ironic-hardware-inventory-recorder-image" ] || [ "$repo" = "metal3-dev-env" ] \
+      || [ "$repo" = "metal3-helm-chart" ] || [ "$repo" = "static-ip-manager-image" ]; then
+      branch='master'
+    elif [ "$repo" = "metal3-io.github.io" ]; then
+      branch='source'
+    else
+      branch='main'
+    fi
```

Comment:
Small tip: in order to slightly improve the maintenance (ie avoid re-touching this file when a repo branch is migrated), instead of the hard-coded checks, what about using something like `git ls-remote` to determine if a branch exists or not? Ie: 

```git ls-remote -q --exit-code --heads https://github.com/metal3-io/$repo main``` 

will return `0` if `main` was found, `2` otherwise. Different branches could be tested (`main`, `master`, `source`) to discover the suitable one

Reply:
@andfasano thanks for the great suggestion, I will take it. Please take a look now.